types/chrome/index.d.ts

```diff
@@ -1980,60 +1980,62 @@ declare namespace chrome {
             visible?: boolean | undefined;
         }
 
-        export interface UpdateProperties {
-            documentUrlPatterns?: string[] | undefined;
-            checked?: boolean | undefined;
-            title?: string | undefined;
-            contexts?: ContextType[] | undefined;
-            /** Optional. @since Chrome 20  */
-            enabled?: boolean | undefined;
-            targetUrlPatterns?: string[] | undefined;
-            onclick?: Function | undefined;
-            /** Optional. Note: You cannot change an item to be a child of one of its own descendants.  */
-            parentId?: number | string;
-            type?: ContextItemType | undefined;
-            /**
-             * Optional.
-             * @since Chrome 62
-             * Whether the item is visible in the menu.
-             */
-            visible?: boolean | undefined;
-        }
+
+        export interface UpdateProperties extends Omit<CreateProperties, 'id'> {}
```

Comment:
Also shims this into the CreateProperties interface since:

- the docs mention that they are the same (except the `id` property)
- the type were currently not compatible with each other